cryptofeed/exchanges/bybit.py

```diff
@@ -102,8 +105,12 @@ async def message_handler(self, msg: str, conn, timestamp: float):
             await self._book(msg, timestamp)
         elif "instrument_info" in msg["topic"]:
             await self._instrument_info(msg, timestamp)
-        elif "order" in msg["topic"]:
+        elif "order" == msg["topic"]:
             await self._order(msg, timestamp)
+        elif "wallet" == msg["topic"]:
+            await self._balance(msg, timestamp)
+        elif "position" == msg["position"]:
```

Comment:
```suggestion
        elif "position" == msg["topic"]:
```
I'm not sure this was tested. Could you check whether position case was tested?

Reply:
Yes, it wasn't tested.. Sorry for such mistake. I planned like I have to test with actual running with nile because there was no such testing code for Bitmex in cryptofeed repository..

Reply:
Okay as we talked at slack, Let's use demo script.